uber/models/attendee.py

```diff
@@ -583,10 +584,14 @@ def is_not_ready_to_checkin(self):
         return None
 
     @property
-    # should be OK
```

Comment:
I didn't delve into the git history to see why this comment was here, but it seemed okay to remove, since at this point it's fair to say that it IS okay rather than just that it should be.

Reply:
Haha, I hesitate every time I see that comment! I'm always like, "I should delete that... but... what if it's important? Hmmm... I wonder why that was put there? Oh well, it should be OK."